warehouse/static/js/warehouse/index.js

```diff
@@ -39,4 +39,22 @@ docReady(() => {
     e.trigger.setAttribute("aria-label", "Copied!");
     e.clearSelection();
   });
+
+  // Get all elements with class "tooltipped" and bind to focousout and
+  // mouseout events. Change the "aria-label" to "original-label" attribute
+  // value.
+  let setOriginalLabel = (element) => {
+    element.setAttribute("aria-label", element.getAttribute("original-label"));
```

Comment:
Instead of using an attribute called ``original-label``, I think it would be better to use one called ``data-original-label``, and then instead of ``element.getAttribute`` you should be able to use (I think) ``element.dataset.originalLabel``.